docs/howto/https/index.adoc

```diff
@@ -0,0 +1,64 @@
+
+[#enabling-https_{context}]
+= Enabling HTTPS
+
+{Thorntail} provides several ways of enabling HTTPS, depending on your use case.
+Here, we describe the most common and most straightforward way.
+
+.Prerequisites
+
+* Keystore in the JKS format. This keystore must include a private key and a corresponding certificate.
++
+--
+If you don't have a keystore, you can create one using `keytool`:
+
+[source,bash,options="nowrap"]
+----
+keytool -genkeypair -keystore my-keystore.jks -storepass my-password -alias my-https-key -keypass my-password -dname CN=localhost -sigalg SHA1withRSA -validity 100 -keyalg RSA -keysize 2048
+----
+
+This keystore is only suitable for testing, as the certificate is self-signed.
```

Comment:
A keystore with a self signed certificate is created (and anything else?). Since the certificate is self signed, the keystore can be used only for testing.

Reply:
"This keystore includes a self-signed certificate, and so it is only suitable for testing." ?